rebasebot/bot.py

```diff
@@ -170,6 +172,70 @@ def _in_excluded_commits(sha: str, exclude_commits: list) -> bool:
     return False
 
 
+def _find_last_rebase_merge_commit(gitwd: git.Repo, source_repo: Repository, ancestry_path_merges) -> Commit:
+    logging.info("Searching for merge commit from previous rebasebot run to identify downstream commits")
+    pygit2_repo = pygit2.Repository(gitwd.working_tree_dir)
+    for merge_line in ancestry_path_merges:
+        sha, commit_message, committer_email = merge_line.split(" || ", 2)
+        logging.info(f"Checking: \"{commit_message}\"")
+        if "openshift-merge-bot[bot]@users.noreply.github.com" in committer_email:
```

Comment:
So this is the committer for the downstream only PRs Merge commits, right?

Reply:
Yes, having the skip avoids extra work and saves a few seconds.

Reply:
Gotcha thanks